packages/schematics/angular/workspace/files/package.json

```diff
@@ -20,28 +20,28 @@
     "@angular/platform-browser": "<%= experimentalAngularNext ? 'next' : latestVersions.Angular %>",
     "@angular/platform-browser-dynamic": "<%= experimentalAngularNext ? 'next' : latestVersions.Angular %>",
     "@angular/router": "<%= experimentalAngularNext ? 'next' : latestVersions.Angular %>",
-    "core-js": "^2.5.4",
+    "core-js": "2.5.7",
     "rxjs": "<%= latestVersions.RxJs %>",
     "zone.js": "<%= latestVersions.ZoneJs %>"
   },
   "devDependencies": {
-    "@angular/cli": "<%= experimentalAngularNext ? 'next' : '~' + version %>",
+    "@angular/cli": "<%= experimentalAngularNext ? 'next' : version %>",
     "@angular/compiler-cli": "<%= experimentalAngularNext ? 'next' : latestVersions.Angular %>",
     "@angular/language-service": "<%= experimentalAngularNext ? 'next' : latestVersions.Angular %>",
-    "@types/jasmine": "~2.8.8",
```

Comment:
I think we should leave the schematic templates out of this first round.  We need an update strategy for these as well as possible coordination with core.